src/status_im/contexts/profile/edit/modal/view.cljs

```diff
@@ -0,0 +1,266 @@
+(ns status-im.contexts.profile.edit.modal.view
+  (:require
+    [clojure.string :as string]
+    [oops.core :as oops]
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [react-native.hooks :as hooks]
+    [react-native.platform :as platform]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im.common.avatar-picture-picker.view :as profile-picture-picker]
+    [status-im.common.events-helper :as events-helper]
+    [status-im.common.validation.profile :as profile-validator]
+    [status-im.constants :as c]
+    [status-im.contexts.profile.edit.modal.events]
+    [status-im.contexts.profile.edit.modal.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [utils.responsiveness :as responsiveness]))
+
+(def scroll-view-height (reagent/atom 0))
+(def content-container-height (reagent/atom 0))
```

Comment:
Should we reset these values to zero when we unmount the component?

Reply:
It's probably too much to ask for the namespace to be rewritten to hooks since it's a large one, but that would be the best since that's the direction we are going.

Reply:
I think I prepared the component to be rewritten easier than before.

I didn't want to touch the implementation too much because these global ratoms are needed to control the floating button, that implementation was very buggy and now seems stable. I think that refactor should be done in a separate PR.